internal/webhook/managedcluster_webhook_test.go

```diff
@@ -28,12 +28,28 @@ import (
 	"github.com/Mirantis/hmc/test/objects/managedcluster"
 	"github.com/Mirantis/hmc/test/objects/management"
 	"github.com/Mirantis/hmc/test/objects/template"
+	chain "github.com/Mirantis/hmc/test/objects/templatechain"
+	tm "github.com/Mirantis/hmc/test/objects/templatemanagement"
 	"github.com/Mirantis/hmc/test/scheme"
 )
 
+type testCase struct {
+	name              string
+	oldManagedCluster *v1alpha1.ManagedCluster
+	newManagedCluster *v1alpha1.ManagedCluster
+	existingObjects   []runtime.Object
+	err               string
+	warnings          admission.Warnings
+}
+
 var (
-	testTemplateName = "template-test"
-	testNamespace    = "test"
+	testNamespace = "test"
+
+	testTemplateName      = "template-test"
+	templateUpgradeSource = "template-1-0-0"
+	templateUpgradeTarget = "template-1-0-1"
+
+	ctChainName = "test"
```

Comment:
i think those may be separated into the `const` section to avoid any accidental data changes